cmd/snap/cmd_snap_op.go

```diff
@@ -763,9 +763,9 @@ func (x *cmdRefresh) listRefresh() error {
 	defer w.Flush()
 
 	// TRANSLATORS: the %s is to insert a filler escape sequence (please keep it flush to the column header, with no extra spaces)
-	fmt.Fprintf(w, i18n.G("Name\tVersion\tRev\tPublisher%s\tNotes\n"), fillerPublisher(esc))
+	fmt.Fprintf(w, i18n.G("Name\tVersion\tRev\tPublisher%s\tNotes\tSize\n"), fillerPublisher(esc))
```

Comment:
I think Notes column should always be last (for consistency with other commands, and also because it can be wide, so it looks better).

Reply:
I left Size for last since changing the order could break scripts using this command but you're right, I'll put Notes last instead. Thanks!

Reply:
While we try not to explicitly break anyone, we don't maintain that the output of our commands is stable. For folks wanting a totally stable response, we have the REST API. But if someone comes back after we change the output and gives a good rationale for why we should undo the change from the CLI output formatting we will consider it (i.e. we changed snap version and then changed it back because it broke a bunch of scripts).

Reply:
I see, that's reasonable. Thanks for explaining that

Reply:
there is some argument for putting Size between Rev and Publisher, because it is tied to revision and so as fast changing, vs publisher and notes that are more stable (usually) for a given snap/channel

Reply:
Makes sense, I'll change that